powerhome/playbook
# [PLAY-1479] Beta Link Kit

**What does this PR do?**

- Create a Link Kit for Rails and React.
- Add color, dark, disabled, icon, icon right, tag, and underline props

[PLAY-1479](https://runway.powerhrg.com/backlog_items/PLAY-1479)

**Screenshots:** Screenshots to visualize your addition/change
![Screenshot 2024-10-25 at 9 52 18 AM](https://github.com/user-attachments/assets/45f0d1af-a340-4558-a9cc-909bbc4a813a)

**How to test?** Steps to confirm the desired behavior:
1. Go to /kits/link/
2. Test Rails and React
3. Compare the handoff to each prop
4. Visit the link (in a non-incognito window). Go back, the link should be purple and "visited"
5. Hover over links to see the color
6. "Focus" on the link by tabbing until you hit the link
7. Use dark mode


#### Checklist:
